cloudevents/pydantic/event.py

```diff
@@ -0,0 +1,240 @@
+#  Copyright 2018-Present The CloudEvents Authors
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+import datetime
+import typing
+
+import pydantic
+
+from cloudevents.pydantic import event_attribute_types
+from cloudevents.pydantic.json_format import ce_json_dumps, ce_json_loads
+from cloudevents.sdk.event import defaults
+
+
+class CloudEvent(pydantic.BaseModel):
+    """
+    Python-friendly cloudevent class supporting v1 events
+    Supports both binary and structured mode CloudEvents
+    """
+
+    data: typing.Optional[typing.Any] = pydantic.Field(
+        title="Event Data",
+        description=(
+            "CloudEvents MAY include domain-specific information about the occurrence. "
+            "When present, this information will be encapsulated within data."
+            "It is encoded into a media format which is specified by the datacontenttype attribute "
+            "(e.g. application/json), and adheres to the dataschema "
+            "format when those respective attributes are present."
+        ),
+    )
+    source: event_attribute_types.NonEmptyURIReference = pydantic.Field(
+        title="Event Source",
+        description=(
+            "Identifies the context in which an event happened. "
+            "Often this will include information such as the type of the event source, "
+            "the organization publishing the event or the process that produced the event. "
+            "The exact syntax and semantics behind the data encoded in the URI is defined by the event producer.\n"
+            "\n"
```

Comment:
Looks like some bad copy-paste? Why do you have `\n` in here?

Reply:
I have the \n for better legibility. Im assuming the documentation will look nicer on the swagger rendering of FastAPI. need to check that

Reply:
yes, it must include the newlines for pretty documentation
![image](https://user-images.githubusercontent.com/3898216/179633922-ed1b5817-334f-42e3-ba48-914c22224973.png)